packages/mdc-checkbox/foundation.ts

```diff
@@ -194,6 +164,7 @@ class MDCCheckboxFoundation extends MDCFoundation {
     } = MDCCheckboxFoundation.cssClasses;
 
     switch (oldState) {
+    // @ts-ignore:no-switch-case-fall-through already existing fallthrough
```

Comment:
Re-enable linter after this `switch()` block
(or use `@tslint-ignore-line` / `@tslint-ignore-next-line` instead).